client/src/components/Form/FormElement.vue

```diff
@@ -43,8 +46,24 @@ export default {
             type: String,
             default: null,
         },
+        type: {
+            type: String,
+            default: "text",
+        },
     },
     computed: {
+        currentValue: {
```

Comment:
Just seeing that this change here, causes a dependency between the element wrapper and the actual type. This needs to be resolved before merging.